_posts/2020-02-21-newpipe-0.18.4-released.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "NewPipe 0.18.4 released: A language selector, improved kodi support and more"
+short: "NewPipe 0.18.4 released"
+date: 2020-02-21 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+It is here, really. Folks, a new minor release of NewPipe: 0.18.4! Bringing you copy-able comments, a kodi-share button, a language selector(!) and many improvements/fixes.
+<!-- more -->
+
+## Language selector
+
+Yes, you did not misread that part. Finally, there is the possibility for you to select the language NewPipe runs in. You can access this setting via <code><nobr><img src="/img/icons/baseline-settings-20px.svg" /> Settings</nobr> > <nobr><img src="/img/icons/baseline-language-20px.svg" /> Content</nobr> > <nobr>App language</nobr></code>. Your selected language will be applied after you restart the app. Big thanks to [@B0pol](https://github.com/B0pol) for implementing this. And while you scroll down the impressive list of available languages, thank in your mind all the amazing people helping translate NewPipe over at [Weblate](https://hosted.weblate.org/projects/newpipe/) and maybe become one of them :)
+
+## Copy comments
+
+Since [v0.16.0]({% link _posts/2019-02-28-newpipe-0.16.0-released.md %}), you can view comments inside the app. Now, roughly a year later and because of the work [@harshlele](https://github.com/harshlele), you can copy them. All you have to do is long press the comment you'd like to share - that's it.
+
+## Kodi button
+
+If you read this heading and thought "Hey. I have this Kodi button enabled in my settings. Do I get another button?" Damn right you get. Presenting the in-player-kodi-button:
+
+<img class="no-flow" src="/img/screenshots/kodi_player_button.jpg"/>
+
+
+## Improvements
+
+Do you know those ReCaptcha images you have to solve when you surf some websites? Some of our users have seen this inside of NewPipe :( While there is no way to let them magically disappear, NewPipe is now able to deal with them properly, thanks to [@Stypoxs](https://github.com/Stypox) developing and [@juanfra684s](https://github.com/juanfra684) testing. [@karkaminski](https://github.com/karkaminski) improved the main view of the app through removing the legacy dot menu next to the magnifying glass. Also included in his changes is the removing of the history sidebar entry if you don't have history enabled (kinda makes sense). 
```

Comment:
```suggestion
Do you know those ReCaptcha images you have to solve when you surf some websites? Some of our users have seen this inside of NewPipe :( While there is no way to let them magically disappear, NewPipe is now able to deal with them properly, thanks to [@Stypox](https://github.com/Stypox) developing and [@juanfra684s](https://github.com/juanfra684) testing. [@karkaminski](https://github.com/karkaminski) improved the main view of the app through removing the legacy dot menu next to the magnifying glass. Also included in his changes is the removing of the history sidebar entry if you don't have history enabled (kinda makes sense). 
```

Reply:
Now you deleted only one s. I think it needs to be there because it shows belonging. It was Stypoxeses developing and Juanfrases testing. Like, I have to put an s there.

Reply:
My bad, I now added two `'`

Reply:
right, that's it!